test/runtime/samples/component-slot-let-scope/_config.js

```diff
@@ -0,0 +1,3 @@
+export default {
+	error: 'thing is not defined'
```

Comment:
why is it an error in this case?

Reply:
Runtime error because thing is not declared anywhere nor is it a global.